creator-node/src/serviceRegistry.js

```diff
@@ -110,11 +110,22 @@ class ServiceRegistry {
     const { queue: sessionExpirationQueue } = this.sessionExpirationQueue
     const { queue: skippedCidsRetryQueue } = this.skippedCIDsRetryQueue
 
+    // Make state machine queues truncate long data (they have jobs with large inputs and outputs)
+    const stateMonitoringAdapter = new BullAdapter(stateMonitoringQueue, {
+      readOnlyMode: true
+    })
+    const stateReconciliationAdapter = new BullAdapter(
+      stateReconciliationQueue,
+      { readOnlyMode: true }
+    )
+    stateMonitoringAdapter.setFormatter('name', this.truncate)
+    stateReconciliationAdapter.setFormatter('name', this.truncate)
```

Comment:
whoops that explains why the dashboard looked the same - fixed and tested locally with a limit of 10 characters (instead of 10,000) to make sure it's really working